### PR TITLE
Ratchet 1 Widescreen and Project Arms 

### PR DESCRIPTION
**Pull request by FrankyBuster**
Changed some stuff in the ratchet pnach to make the widescreen patch function, and ported Project Arms pnatch from OPL

### CODE REVIEW

File: patches/SCUS-97199_CE4933D0.pnach [hunk 1]
```diff
@@ -5,97 +5,97 @@
 //comment=Widescreen hack by PsxFan107. Currently it appears to break rendering of textures in some areas.
```
Comment (pgert) [inline patches/SCUS-97199_CE4933D0.pnach:5]: "Currently it appears to break rendering of textures in some areas."
Is this now only a minor problem?
If it's no longer a problem, then delete the sentence.

File: patches/SLPS-25102_49D8C868.pnach [hunk 2]
```diff
@@ -0,0 +1,16 @@
+gametitle=Project Arms (J) (SLPS_251.02)
+
+[Widescreen 16:9]
+gsaspectratio=16:9
+comment=Widescreen hack by Arapapa
```
Comment (pgert) [inline patches/SLPS-25102_49D8C868.pnach:5]: author=Arapapa, ported by FrankyBuster
comment=Widescreen hack

File: patches/SCUS-97199_CE4933D0.pnach [hunk 3]
```diff
@@ -2,100 +2,100 @@
 
 //[Widescreen 16:9]
 //gsaspectratio=16:9
-//comment=Widescreen hack by PsxFan107. Currently it appears to break rendering of textures in some areas.
+//comment=Widescreen hack by PsxFan107.
```
Comment (pgert) [inline patches/SCUS-97199_CE4933D0.pnach:5]: You need to enable this:
[Widescreen 16:9]
gsaspectratio=16:9

And while at it, change "//comment=Widescreen hack by PsxFan107." into:
author=PsxFan107
comment=Widescreen hack

**Comment by refractionpcsx2** [comment]
All you seem to have done is re-enable the patch and needlessly change the extendeds to words (which doesn't change anything)

Have you done anything to fix this patch?

**Pull request closed without merging**